tests/ui/useless_attribute.fixed

```diff
@@ -0,0 +1,49 @@
+// run-rustfix
+// aux-build:proc_macro_derive.rs
+
+#![warn(clippy::useless_attribute)]
+#![warn(unreachable_pub)]
+
+#![allow(dead_code)]
+#![cfg_attr(feature = "cargo-clippy", allow(dead_code))]
+#[rustfmt::skip]
+#[cfg_attr(feature = "cargo-clippy",
+           allow(dead_code))]
+#[allow(unused_imports)]
+#[allow(unused_extern_crates)]
+#[macro_use]
+extern crate clippy_lints;
+
+#[macro_use]
+extern crate proc_macro_derive;
```

Comment:
Would it also emit the lint, when you add `#[cfg_attr(..)]` here?

Reply:
Not this particular case, because `#[macro_use]` with `extern crate` is whitelisted. It will trigger if it's changed like in the other diff, though. Both cases would be fixed by checking for previous items.